spec/forms/steps/client/has_nino_form_spec.rb

```diff
@@ -13,12 +13,14 @@
     }
   end
 
-  let(:crime_application) do
-    instance_double(CrimeApplication)
-  end
+  let(:crime_application) { instance_double(CrimeApplication, applicant: 'applicant') }
 
   let(:nino) { nil }
 
+  before do
```

Comment:
We can remove this now, as we don't have this call on the NINO form object anymore 😄